readme.md

```diff
@@ -69,6 +69,8 @@ Watch task helps you reduce your efforts during development extensions. If the t
 gulp watch
 ```
 
+In order for Live-reload to work, reload extension manually after starting `gulp watch`.
```

Comment:
`reload` is not only option to load an extension. Could you add a specific condition that the extension is already loaded before watch task?

Reply:
And BTW: If you are like me that you want to see your changes immediately reflected every 2 or 3 seconds (could be because you are coding fast or debugging) then I suggest to change reload time to 2 seconds which would be 200 or 3 seconds which would be 300) in https://github.com/yeoman/generator-chrome-extension/blob/master/app/templates/scripts/chromereload.js#L54